dlt/normalize/items_normalizers.py

```diff
@@ -112,23 +118,96 @@ def __call__(
 
 
 class ParquetItemsNormalizer(ItemsNormalizer):
+    RECORD_BATCH_SIZE = 100000
```

Comment:
we need to do something with this. maybe we could use `https://arrow.apache.org/docs/python/generated/pyarrow.parquet.ParquetFile.html#pyarrow.parquet.ParquetFile.read_row_groups` instead of reading batches? then we could configure the number of groups to read. it also could be faster because the method returns tables, not batches

Reply:
Yeah that's probably better. It's using the chunk size the table is written with originally so should be more efficient.

Reply:
Makes no difference in speed from my tests if we're using the same batch size/row group size. But still makes sense to use this I think so we're rewriting the file with the same chunk size and not changing it.